toolchain/BUILD.bazel

```diff
@@ -101,18 +102,54 @@ cc_args_list(
     }),
 )
 
+cc_args_list(
+    name = "sysroot_flags",
+    args = select({
+        "@platforms//os:macos": [
+            "//toolchain/args/macos:macos_sdk_sysroot",
+        ],
+        "//conditions:default": [
+            "//toolchain/args:empty_sysroot_flags",
+        ],
+    }),
+    visibility = ["//toolchain:__subpackages__"],
```

Comment:
should we makethis default package vis?

Reply:
We can't because of config_settings just yet. but yes we should aim at that.